protocols/CopyToResultsDir.sh

```diff
@@ -197,12 +197,10 @@ rm -f "${projectResultsDir}/rawdata/ngs/"*".phiX.recoded.fq.gz"
 echo "pipeline is finished"
 
 runNumber=$(basename "$(dirname "${projectResultsDir}")")
-if [ -f "${logsDir}/${project}/${runNumber}.pipeline.started" ]
-then
-	mv "${logsDir}/${project}/${runNumber}.pipeline".{started,finished}
-else
-	touch "${logsDir}/${project}/${runNumber}.pipeline.finished"
-fi
+
+rm -f "${logsDir}/${project}/${runNumber}.pipeline.started"
+touch "${logsDir}/${project}/${runNumber}.pipeline.finished"
```

Comment:
It was so confusing to have the timestamp of the .started on the .finished file

Reply:
`rm` + `touch` is not a single atomic operation. If you have issues with the timestamp, then you can update  the timestamp by moving the touch out of the `if`:
```
if [ -f "${logsDir}/${project}/${runNumber}.pipeline.started" ]
then
	mv "${logsDir}/${project}/${runNumber}.pipeline".{started,finished}
fi
touch "${logsDir}/${project}/${runNumber}.pipeline.finished"
```